ksql-engine/src/main/java/io/confluent/ksql/KsqlContext.java

```diff
@@ -84,8 +85,16 @@ public void sql(final String sql, final Map<String, Object> overriddenProperties
     }
   }
 
+  /**
+   * @deprecated use {@link #getPersistentQueries}.
+   */
+  @Deprecated
```

Comment:
What's the thinking behind marking a method as deprecated vs. simply removing it? Why do we prefer to deprecate this?

Reply:
According to @apurvam this is a public API that people may be using for KSQL in 'embedded mode'.  As such, I didn't want to change the existing method as it could break customers code.